pkg/ccl/backupccl/backup_processor.go

```diff
@@ -247,9 +239,9 @@ type spanAndTime struct {
 	lastTried  time.Time
 }
 
-type returnedSST struct {
-	f              BackupManifest_File
-	sst            []byte
+type returnedSpans struct {
```

Comment:
I think this should be singular?

Reply:
maybe should be called "exportedSpan" actually; "returnedSpan" just sounds like somebody returned a roachpb.Span, but this is much more than that; it is the actual data from that span.

Reply:
Changed, but you sure about singular? There's literally a field called `completedSpans` in the object.